src/llvm/analysis/ReachingDefinitions/ReachingDefinitions.cpp

```diff
@@ -772,7 +772,8 @@ LLVMRDBuilder::createCall(const llvm::Instruction *Inst, RDBlock *rb)
 
```

Comment:
This should be merged to 59ecad4ef086 (and commented in the commit message)

Reply:
Actually, in the `HEAD` commit, uses for call nodes are not added. I believe it would be even better to drop 59ecad4, e7dd946 and 0788f15.